core/vm/logger.go

```diff
@@ -76,6 +77,7 @@ type StructLogger struct {
 	cfg LogConfig
 
 	logs          []StructLog
+	sizeLogs      int
```

Comment:
I don't know why this portion is not yet marked `outdated`.  From the `Files Changed` link above, I no longer see this diff.